### DISCUSSION TITLE
模板中 类修饰符 x.cs_class_modifier 如何指定？

### DISCUSSION THREAD

**Discussion by cchorseless** [focus-creative-games/luban#6] [Q&A]
模板中 类修饰符 x.cs_class_modifier 如何指定？

**Comment by cchorseless** [comment]
备注  unity  生成的代码文件需要

**Comment by pirunxi** [comment]
没太懂你的意思，x.cs_class_modifier 返回 abstract 之类的修饰符，跟你备注unity生成没啥关系。 有更多疑问请进群
